### PR TITLE
Update RPI docs on installing openssl 1.0.2

### PR DESCRIPTION
**Pull request by dilin-MS**

### CODE REVIEW

File: docs/embedded-linux/embedded-linux-get-started.md [hunk 1]
```diff
@@ -60,8 +60,19 @@ This tutorial shows how to develop and compile a simple device app written in C
...
+    ```bash
+    apt-get install libssl1.0 linssl1.0-dev
+    sudo ln -s /usr/lib/arm-linux-gnueabihf/libssl.so.1.0.2 /usr/lib/libssl.so.1.0.0
+    sudo ln -s /usr/lib/arm-linux-gnueabihf/libcrypto.so.1.0.2 /usr/lib/libcrypto.so.1.0.0
+    ```
+
+    > We use Azure IoT C SDK via Vcpkg, while currently it only supports `openssl 1.0.2`. `openssl 1.1.1` is not supported yet. See [issue 1341](https://github.com/Azure/azure-iot-sdk-c/issues/1341) and [issue 4267](https://github.com/microsoft/vcpkg/issues/4267).
+
+
+4. Run `azure_iot_app [connection string]` and you see the Raspberry Pi start sending telemetry data to the Azure IoT Hub:
```
Comment (dooriya) [inline docs/embedded-linux/embedded-linux-get-started.md]: device connection string

File: docs/embedded-linux/embedded-linux-get-started.md [hunk 2]
```diff
@@ -60,8 +60,19 @@ This tutorial shows how to develop and compile a simple device app written in C
...
-3. Run `azure_iot_app [connection string]` and you see the Raspberry Pi start sending telemetry data to the Azure IoT Hub:
+3. Run the below commands to enable use to `openssl 1.0.2` on Raspberry Pi:
+
+    ```bash
+    apt-get install libssl1.0 linssl1.0-dev
+    sudo ln -s /usr/lib/arm-linux-gnueabihf/libssl.so.1.0.2 /usr/lib/libssl.so.1.0.0
+    sudo ln -s /usr/lib/arm-linux-gnueabihf/libcrypto.so.1.0.2 /usr/lib/libcrypto.so.1.0.0
+    ```
+
+    > We use Azure IoT C SDK via Vcpkg, while currently it only supports `openssl 1.0.2`. `openssl 1.1.1` is not supported yet. See [issue 1341](https://github.com/Azure/azure-iot-sdk-c/issues/1341) and [issue 4267](https://github.com/microsoft/vcpkg/issues/4267).
```
Comment (dooriya) [inline docs/embedded-linux/embedded-linux-get-started.md]: Azure IoT Device C SDK

File: docs/embedded-linux/embedded-linux-get-started.md (hunk 1 above)
Comment (dilin-MS) [reply docs/embedded-linux/embedded-linux-get-started.md]: Fixed

File: docs/embedded-linux/embedded-linux-get-started.md (hunk 2 above)
Comment (dilin-MS) [reply docs/embedded-linux/embedded-linux-get-started.md]: Fixed.

**Pull request merged**